### ISSUE TITLE
ESM not supported

### ISSUE THREAD

**Issue by ctrlaltdylan** [deepgram/deepgram-node-sdk#203]
## What is the current behavior?

Not able to import the Deepgram named export in an ESM project.

```
import { Deepgram } from "@deepgram/sdk";
         ^^^^^^^^
SyntaxError: The requested module '@deepgram/sdk' does not provide an export named 'Deepgram'
```

> What's happening that seems wrong?

Should be able to import with ESM.

## Steps to reproduce

Start a new Node.js project, with ESM enabled, install `@deegram/sdk` and attempt to import the package.

> To make it faster to diagnose the root problem. Tell us how can we reproduce the bug.

## Expected behavior

> What would you expect to happen when following the steps above?

That I can use the `Deepgram` named export in an ESM project.

## Please tell us about your environment

Testing with a Pipedream workflow, which uses ESM by default.

**Comment by lukeocodes** [comment]
hi, check out the v3.0.0-beta.4 tag, with our latest build of the new SDK. you'll find it has ESM support. the interface is different, and a migration guide is already live on our docs

**Comment by ctrlaltdylan** [comment]
Thanks for the quick reply @lukeocodes 

I just tried the combinations of:

* `@deepgram/sdk@3.0.0-beta.4`
* `@deepgram/sdk@3.0.0-beta.3`
* `@deepgram/sdk@3.0.0-beta.2`

And they all gave me the same issue: 

```
The requested module '@deepgram/sdk@3.0.0-beta.{{version}}' does not provide an export named 'Deepgram'
```

I'm referring to the versions in the NPM page: https://www.npmjs.com/package/@deepgram/sdk?activeTab=versions

**Comment by ctrlaltdylan** [comment]
I was able to figure this out, there's a difference between v2 and v3 client instantiation. But I ended up using an HTTP client directly, because the `callback` wasn't a valid option to the v3 version, and I couldn't find documentation on how to pass other arguments besides the `url`.

**Comment by lukeocodes** [comment]
Apologies, I had alluded to our migration guide but I forgot to link it! https://developers.deepgram.com/docs/js-sdk-v2-to-v3-migration-guide

**Comment by ctrlaltdylan** [comment]
All good, thanks @lukeocodes !

**Comment by bplenderleith** [comment]
> I was able to figure this out, there's a difference between v2 and v3 client instantiation. But I ended up using an HTTP client directly, because the `callback` wasn't a valid option to the v3 version, and I couldn't find documentation on how to pass other arguments besides the `url`.

@ctrlaltdylan any chance you could elaborate on what you mean by you ended up using an HTTP client directly? I'm having the same issue with a Pipedream workflow and am at a bit of a loss.

**Comment by lukeocodes** [comment]
@bplenderleith and @ctrlaltdylan I updated my last message about the callback support. It is supported, albeit in separate methods now. I will update the migration guide for this (and add the migration guide link to the readme).

Thanks for your patience!

**Comment by ctrlaltdylan** [comment]
@bplenderleith sure!

Here's the Pipedream compatible Node.js code for use in Node.js code steps or pre-built actions:

```javascript
import { createClient } from '@deepgram/sdk@3.0.0-beta.4'

export default defineComponent({
  props: {
    deepgram: {
      type: "app",
      app: "deepgram",
    },
    media_url: {
       type: "string",
       label: "URL of the media to transcribe"
    }
  },
  async run({steps, $}) {
    const deepgram = createClient(this.deepgram.$auth.api_key);

    const { resume_url, cancel_url } = $.flow.suspend()

    const response = await deepgram.listen.prerecorded.transcribeUrl(
      {
        url: this.media_url,
        callback: resume_url,
      },
      {
        model: "nova",
      }
    );

    return response

  },
}
```

Please note the `callback` param to the `deepgram.listen.prerecorded.transcribeUrl` doesn't current work. Just need @lukeocodes documentation on how to properly pass the `callback` URL.

Without the `callback` URL property, most likely you'll hit a `TIMEOUT` error on longer media. Otherwise, the `$.flow.suspend` works great.

Here's my "vanilla" JS version instead that doesn't use the SDK and the `callback` method _does_ work:

```javascript
import { axios } from "@pipedream/platform"
export default defineComponent({
  props: {
    deepgram: {
      type: "app",
      app: "deepgram",
    },
    media_url: {
       type: "string",
       label: "URL of the media to transcribe"
    }
  },
  async run({steps, $}) {
    const { resume_url, cancel_url } = $.flow.suspend();

    return await axios($, {
      url: `https://api.deepgram.com/v1/listen`,
      method: 'POST',
      data: {
        url: this.media_url,
      },
      params: {
        model: 'nova',
        callback: resume_url
      },
      headers: {
        "Authorization": `Token ${this.deepgram.$auth.api_key}`,
      },
    })
  },
})

```

Also once this [PR is merged into Pipedream](https://github.com/PipedreamHQ/pipedream/pull/9044), then the [**Deepgram - Create Transcription**](https://pipedream.com/apps/deepgram/actions/transcribe-audio) action will use `$.flow.suspend` natively too so you can bypass this timeout problem.


Site note for you @lukeocodes, at Pipedream we're releasing a new feature that allows easy new file uploads, and I've built a really useful workflow as a demo with Deepgram to transcribe our new YouTube videos and update the descriptions of the videos with the Deepgram created transcription.

Are you or anyone at Deepgram interested in doing video together on how this integration works?

**Comment by bplenderleith** [comment]
Thanks a lot @ctrlaltdylan!
